admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/TitleServiceImpl.java

```diff
@@ -83,6 +87,23 @@
 	
 	@Autowired
 	private MasterdataCreationUtil masterdataCreationUtil;
+	
+
+	@Value("${mosip.kernel.masterdata.title_event:masterdata/idauthentication_templates}")
+	private String topic;
```

Comment:
The default value of topic should be: masterdata/titles .. it is not common for all modules.. not specific to ID Authentication.

Reply:
sure sorry

Reply:
Fine